starknet_py/constants.py

```diff
@@ -1,11 +1,29 @@
-# Address came from starkware-libs/starknet-addresses repository: https://github.com/starkware-libs/starknet-addresses
-FEE_CONTRACT_ADDRESS = (
-    "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"
-)
+# pylint: disable=line-too-long
+# fmt: off
```

Comment:
Why is formatting disabled here?

Reply:
To have long integer constants in one line instead this:
```
DEFAULT_DEPLOYER_ADDRESS = (
    "0x041a78e741e5aF2fEc34B695679bC6891742439f7AFB8484Ecd7766661aD02BF"
)
```